dexlib2/src/main/java/com/android/tools/smali/dexlib2/analysis/ArrayProto.java

```diff
@@ -141,7 +140,7 @@ public TypeProto getCommonSuperclass(@Nonnull TypeProto other) {
         return other.getCommonSuperclass(this);
     }
 
-    private static final String BRACKETS = Strings.repeat("[", 256);
+    private static final String BRACKETS = "[".repeat(256);
```

Comment:
[`String.repeat`](https://docs.oracle.com/en%2Fjava%2Fjavase%2F11%2Fdocs%2Fapi%2F%2F/java.base/java/lang/String.html#repeat(int)) was not added until JDK-11.

Can we add a local `StringUtil` implementation of this? Maybe just copy the Guava optimized implementation, https://github.com/google/guava/blob/master/guava/src/com/google/common/base/Strings.java#L152, or as this is the only use just a simple loop here.

Reply:
Ported into StringUtils